lib/kdump_utils.pm

```diff
@@ -121,18 +121,18 @@ sub activate_kdump {
     zypper_call('rm -y ruby2.1-rubygem-nokogiri', exitcode => [0, 104]);
     # get kdump memory size bsc#1161421
     my $memory_total = script_output('kdumptool  calibrate | awk \'/Total:/ {print $2}\'');
-    my $memory_kdump = $memory_total >= 2048 ? 1024 : 640;
+    my $memory_kdump = $memory_total >= 2048 ? 1024 : 320;
```

Comment:
This change should not be mixed with "Add softfail … on aarch64", better split out

Reply:
And I think that 320 is to small for ppc64le, please do verification for this arch  https://openqa.suse.de/tests/4031133#

Reply:
> And I think that 320 is to small for ppc64le, please do verification for this arch https://openqa.suse.de/tests/4031133#

please check already provided verification:
https://openqa.suse.de/tests/4028977#next_previous (QEMURAM 1024 ppc64le)

Reply:
> This change should not be mixed with "Add softfail … on aarch64", better split out

I don't see the real reason for split out here. If this is not working, yes, I should handle ppc64le and aarch64 differently. but please look at verification runs. I checked them and tried with different sizes.

Reply:
320 MB is safe and works well for both when QEMURAM=1024.

Reply:
Ok, thanks for the verification.